include/wally_elements.h

```diff
@@ -17,15 +17,20 @@ extern "C" {
 
 #define ASSET_COMMITMENT_LEN 33 /** Length of an Asset Value Commitment */
 
-#define ASSET_RANGEPROOF_MAX_LEN 5134 /** Maximum length of an Asset Range Proof */
+#define ASSET_RANGEPROOF_MAX_LEN 5134 /** Maximum length of an Asset Value Range Proof */
+#define ASSET_EXPLICIT_RANGEPROOF_MAX_LEN 73 /** Maximum length of an Explicit Asset Value Range Proof */
```

Comment:
Is this the max length or the exact length?

in `src/elements.c` we have both
```
        if (asset || asset_len || abf || abf_len || extra || extra_len ||
            len < ASSET_EXPLICIT_RANGEPROOF_MAX_LEN)
            goto cleanup;
            
        ...
        
        if (!rangeproof || rangeproof_len > ASSET_EXPLICIT_RANGEPROOF_MAX_LEN ||
            parse_commitment(ctx, commitment, commitment_len, &commit) != WALLY_OK ||
            parse_generator(ctx, generator, generator_len, &gen) != WALLY_OK)
            return WALLY_EINVAL;
```

Reply:
It's the max length, zero values can be slightly shorter. So these checks are correct: the first requires the caller pass a sufficient buffer for the largest possible explicit proof, and the second ensures that the passed proof is not trivially invalid (e.g. a full rangeproof).